cita-executor/evm/Cargo.toml

```diff
@@ -16,6 +16,9 @@ rlp = { git = "https://github.com/cryptape/cita-common.git", branch = "develop"
 rustc-hex = "1.0"
 db = { git = "https://github.com/cryptape/cita-common.git", branch = "develop" }
 
+cita-vm = { git = "https://github.com/cryptape/cita-vm", branch = "cita" }
```

Comment:
Can you delete this two dependencies now?

Reply:
After taken `storage.rs` out from evm, It can be done!

Reply:
But you have done it in this PR?

Reply:
Yes, your pr merge firstly. Some works to be done!